docs/src/main/sphinx/installation/cli.rst

```diff
@@ -160,6 +160,14 @@ press :kbd:`Enter`.
 By default, you can locate the Trino history file in ``~/.trino_history``.
 Use the ``TRINO_HISTORY_FILE`` environment variable to change the default.
 
+Autosuggestion
+--------------
+As you type commands in CLI , you will see a completion offered after the cursor
```

Comment:
... in the CLI, you see a suggested completion offered ...

[no future tense in docs]